docs/source/pythonapi/examples/mgxs-part-iii.ipynb

```diff
@@ -1108,7 +1104,7 @@
      "name": "stderr",
      "output_type": "stream",
      "text": [
-      "/home/romano/openmc/openmc/tallies.py:1506: RuntimeWarning: invalid value encountered in true_divide\n",
+      "/home/nelsonag/git/openmc/openmc/tallies.py:1506: RuntimeWarning: invalid value encountered in true_divide\n",
```

Comment:
It doesn't appear you've run this notebook to completion, right? I would have expected that you'd have to set `nu` somehow on the `Library` object. In addition, I'm not sure that this will work with the `openmoc.materialize` module since it currently requires a `'nu-fission'` MGXS type, though that can be easily fixed if this PR is to be merged.